sonic-net/sonic-buildimage
# New NAC Container addition and related source support for development of PNAC feature for HLD PR 1292

… of PNAC feature for HLD PR 1292



#### Why I did it
Development changes including new NAC container addition and related source inclusion for HLD PR 1292.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
1. New container NAC is included in SONiC NOS build framework
2. Modified corresponding makefiles and configuration files to run the new container
3. Build framework update for generating new hostapd debian package and inclusion into docker-nac

#### How to verify it
Generate SONiC build image with NAC feature enabled and verify using docker commands.
Backporting not required. These are the changes of new feature PNAC - HLD PR 1292. 

Sample output:
root@sonic:~# docker ps
CONTAINER ID   IMAGE                             COMMAND                  CREATED        STATUS              PORTS     NAMES
**016669fad085   28648a1c5833                      "/usr/local/bin/supe…"   21 hours ago   Up 48 seconds                 nac**
ea69933634ce   docker-lldp:latest                "/usr/bin/docker-lld…"   21 hours ago   Up 49 seconds                 lldp
a57e1225225d   docker-router-advertiser:latest   "/usr/bin/docker-ini…"   21 hours ago   Up 48 seconds                 radv
e7abf1901bda   docker-gbsyncd-vs:latest          "/usr/local/bin/supe…"   21 hours ago   Up 51 seconds                 gbsyncd
29416969fcd1   docker-syncd-vs:latest            "/usr/local/bin/supe…"   21 hours ago   Up 52 seconds                 syncd
91fa4d690825   docker-teamd:latest               "/usr/local/bin/supe…"   21 hours ago   Up 52 seconds                 teamd
1faf3091d7db   docker-orchagent:latest           "/usr/bin/docker-ini…"   21 hours ago   Up 55 seconds                 swss
44c8450cb015   docker-fpm-frr:latest             "/usr/bin/docker_ini…"   21 hours ago   Up 56 seconds                 bgp
3ea5bb4b64ef   docker-database:latest            "/usr/local/bin/dock…"   21 hours ago   Up About a minute             database


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
